centos.sh

```diff
@@ -102,49 +104,49 @@ generate_new_ramdisk() {
     # pick the latest kernel
     VERSION="$(ls -r -1 $FOLD/hdd/boot/vmlinuz-* | head -n1 |cut -d '-' -f 2-)"
 
-    if [ $IMG_VERSION -lt 60 ] ; then
+    if [ "$IMG_VERSION" -lt 60 ] ; then
       MODULESFILE="$FOLD/hdd/etc/modprobe.conf"
       # previously we added an alias for eth0 based on the niclist (static
       # pci-id->driver mapping) of the old rescue. But the new rescue mdev/udev
       # So we only add aliases for the controller
-      echo -e "### Hetzner Online GmbH - installimage" > $MODULESFILE 2>>$DEBUGFILE
-      echo -e "# load all modules" >> $MODULESFILE 2>>$DEBUGFILE
-      echo -e "" >> $MODULESFILE 2>>$DEBUGFILE
+      echo -e "### $COMPANY - installimage" > "$MODULESFILE" 2>> "$DEBUGFILE"
+      echo -e "# load all modules" >> "$MODULESFILE" 2>> "$DEBUGFILE"
+      echo -e "" >> "$MODULESFILE" 2>> "$DEBUGFILE"
 
-      echo -e "# hdds" >> $MODULESFILE 2>>$DEBUGFILE
+      echo -e "# hdds" >> "$MODULESFILE" 2>> "$DEBUGFILE"
       HDDDEV=""
       for hddmodule in $MODULES; do
```

Comment:
Where comes $MODULES from? Is $MODULES an array? It should really be an array, because iterating over a space delimted string is dangerous.